CONTRIBUTING.md

```diff
@@ -74,7 +74,10 @@ cp -r basic-sqlite my-repro && cd my-repro
 pnpm install
 # Ensure that the db and the schema are synced
 pnpm dbpush
-# Do some code changes, always re-generate the client, then try it out
+# Edit the desired package and build it
+# (or similar command, see above)
+pnpm -r run build 
```

Comment:
I think this is confusing here.
Because you want to say to run this in the root directory, right? Here we are in `./sandbox`

Reply:
Also `pnpm -w run build` is enough and can be run from almost anywhere (except `./sandbox` because it has a `pnpm-workspace.yaml`
`-w` means the command will run from the root directory, and it is configured as `"build": "pnpm -r build",`